### PR TITLE
add 2 new agent names in html file and styles in css file plus a pend…

### PR DESCRIPTION
**Pull request by Kazathur92**
…ing new mission on wow.html

### CODE REVIEW

**Review by CurtainUp — APPROVED** [review]
Fixed a couple spelling errors. 👍

File: secretmission.html [hunk 1]
```diff
@@ -9,11 +9,19 @@
 </head>
 <body>
   <section class="mission"></section>
-    <h1>Secret Mission</h1>
+    <h1>New Secret Mission</h1>
     <h2>THIS INFORMATION IS CLASSIFIED</h2>
     <p class="hidden">
-      Give Elyse and Alfonso a high five because they're awesome!
+      We have a new agent, name is Bryan. Hes the deadliest baddest and craziest operative we have ever worked with.
```
Comment (BryanNilsen) [inline secretmission.html:15]: This is excellent! I am deadly and badass.

**Comment by BryanNilsen** [comment]
👍

**Review by BryanNilsen — APPROVED** [review]
👍

**Comment by noahwoods** [comment]
🦆

**Pull request merged**